docs/changes.rst

```diff
@@ -134,6 +134,76 @@ Changelog
 Version 1.1.0rc1 (unreleased)
 -----------------------------
 
+Compatibility notes:
+
+- delete: removed short option for --cache-only
+
+New features:
+
+- support borg list repo --format {comment} {bcomment} {end}, #2081
+- key import: allow reading from stdin, #2760
+
+Fixes:
+
+- use commit() in borg with-lock, #1867
+- prune: fix checkpoints processing with --glob-archives
+- FUSE: versions view: keep original file extension at end, #2769
+- fix --last, --first: do not accept values <= 0,
+  fix reversed archive ordering with --last
+- include attic.tar.gz when installing the package
+
+Other changes:
+
+- FUSE: versions view: linear numbering by archive time
+- use limited unpacker for outer key, for manifest, #2174 #2175
+- split up interval parsing from filtering for --keep-within, #2610
+- add a basic .editorconfig, #2734
+- docs:
+
+  - better explanation of the --ignore-inode option relevance, #2800
+  - fix openSUSE command and add openSUSE section
+  - simplify ssh authorized_keys file using "restrict", add legacy note, #2121
+  - mount: show usage of archive filters
+  - mount: add repository example, #2462
+  - info: update and add examples, #2765
+  - prune: include example
+  - html: include group description in output
+  - no third level toc on command usage pages ("Description", "Examples")
+  - use DOM ready event, not document loaded event
+  - various formatting fixes
+  - use tables for options, with even widths.
+  - retain rST option list formatting (for Common Options)
+  - skip empty option groups
+  - uniform tables for listing options (HTML only)
+- code:
+
+  - hashindex: don't pass side effect into macro
+  - crypto low_level: don't mutate local bytes()
+  - use dash_open function to open file or "-" for stdin/stdout
+  - archiver: argparse cleanup / refactoring
+  - shellpattern: add match_end arg
+- tests:
+
+  - patching objects does not work with the borg.exe binary, #2700
+  - remove skipping the noatime tests on GNU/Hurd, #2710
+  - xattr: test split_lstring
+  - version: add missing test for format_version, fix bug
+  - archiver:
+
+    - add test for "create -" and "extract --stdout"
+    - add test_benchmark_crud, fix bug
+    - add tests for debug info / refcount-obj
+    - add test for paperkey import, fix bug
+- vagrant:
+
+  - fix set-up of cygwin and its sshd
+  - add Debian 9 "stretch" 64bit
+- travis:
+
+  - don't perform full travis build on docs-only changes, #2531
+  - install fakeroot for Linux (was removed from preinstalled sw stack)
+
```

Comment:
A more general question, the changelogs are always quite large, so is it worth it to include things like CI or vagrant changes, or mentioning that a single unit test was added? (Or the many items under "docs:" that basically mean "the formatting changed a bit")

For example, I'd just summarize the `tests:` section with "Added a number of additional unit tests". I doubt anyone cares for more details in the changelog.

Stuff like CI changes are basically unrelated to releases as well.